include/geometric_shapes/bodies.h

```diff
@@ -225,8 +225,7 @@ class Body
      Sometimes multiple attempts need to be generated.
      The function terminates with failure (returns false) after \e max_attempts attempts.
      If the call is successful (returns true) the point is written to \e result */
-  virtual bool samplePointInside(random_numbers::RandomNumberGenerator& rng, unsigned int max_attempts,
-                                 Eigen::Vector3d& result) const;
+  virtual bool samplePointInside(unsigned int max_attempts, Eigen::Vector3d& result) const;
```

Comment:
I'm happy to see these new methods for simplified use, but please keep the option to specify a seeded RNG.
MoveIt is hard enough to use deterministically already.

Reply:
Thanks for this feedback, I changed the API again to give the user the option to pass custom random values to these functions by passing a lambda that generates random values. This way the interface is flexible and we don't need to pass a library-specific RandomNumberGenerator that might not cover the functionality the user wishes. I'd like to hear your opinion on this